salt/states/pkg.py

```diff
@@ -1053,8 +1053,16 @@ def installed(
     if not isinstance(version, six.string_types) and version is not None:
         version = str(version)
 
+    refreshed = False
```

Comment:
@terminalmage, would you be ok with "was_refreshed"? That way when we do the test we'll end up seeing
```
if os.path.isfile(rtag) and was_refreshed:
```
I think that "was_refreshed" captures the meaning a bit more. However, if you still insist on "already_refreshed" I'll go with that, as I said.